### PR TITLE
Add jsonrpcclient recipe

### PR DESCRIPTION
**Pull request by bcb**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/jsonrpcclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f2c5d6bed0d37e4f1f7e76c6e0ae2669ef2facd709508398cc54086b83a6dcf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/jsonrpcclient/meta.yaml]: I think it could be ``noarch``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

Comment (marcelotrevisani) [reply recipes/jsonrpcclient/meta.yaml]: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/jsonrpcclient/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  noarch: python
+
+requirements:
+  host:
+    - python3
```
Comment (marcelotrevisani) [inline recipes/jsonrpcclient/meta.yaml]:
```suggestion
    - python >3
```

File: recipes/jsonrpcclient/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - pip
+  run:
+    - python3
```
Comment (marcelotrevisani) [inline recipes/jsonrpcclient/meta.yaml]:
```suggestion
    - python >3
```

File: recipes/jsonrpcclient/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - jsonrpcclient
+
+about:
+  home: https://github.com/bcb/jsonrpcclient
+  license: MIT
+  license_family: MIT
+  license_file: https://github.com/bcb/jsonrpcclient/blob/master/LICENSE
```
Comment (marcelotrevisani) [inline recipes/jsonrpcclient/meta.yaml]: https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

File: recipes/jsonrpcclient/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  sha256: 6f2c5d6bed0d37e4f1f7e76c6e0ae2669ef2facd709508398cc54086b83a6dcf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >3
```
Comment (scopatz) [inline recipes/jsonrpcclient/meta.yaml]: Should probably be `python >=3` here and below

File: recipes/jsonrpcclient/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c50860409b73af9f94b648439caae3b4af80d5ac937f2a8ac7783de3d1050ba9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/jsonrpcclient/meta.yaml]: Please, add the ``entry_points``
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - jsonrpc=jsonrpcclient.__main__:main
```

**Comment by bcb** [comment]
All checks are passing.

File: recipes/jsonrpcclient/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    - jsonrpc=jsonrpcclient.__main__:main
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - apply_defaults <1
+    - click <7
+    - jsonschema <4
```
Comment (marcelotrevisani) [inline recipes/jsonrpcclient/meta.yaml]: I think you just need ``pip`` and ``python`` here in host. Could you please try that?

Comment (bcb) [reply recipes/jsonrpcclient/meta.yaml]: Pushed the change, checks are in progress.

Comment (bcb) [reply recipes/jsonrpcclient/meta.yaml]: Checks passed.

**Pull request merged**